docs/specs/moniker.yml

```diff
@@ -1808,3 +1808,184 @@ inputs:
 outputs:
   136a42ac/docs/a.json: |
     {"metadata":{"monikers":["netcore-1.0"]}}
+---
+# Compatible for group in docfx.json of v2
+repos:
+  https://github.com/ops/config-1:
+  - files:
+      .openpublishing.publish.config.json: |
```

Comment:
consider creating a minimal test spec, `.openpublishing.publish.config.json` is not needed? Same as other test cases.

Reply:
sure, thought we need to have `.openpublishing.publish.config.json` along with v2 `docfx.json`